src/sas/qtgui/Perspectives/Fitting/media/fitting_help.rst

```diff
@@ -650,6 +650,41 @@ The DREAM optimiser is the most sophisticated, but may not necessarily be the be
 option for fitting simple models. If uncertain, try the Levenberg-Marquardt optimiser
 initially.
 
+Some model parameters, for example, radii/lengths or orientation angles can be
+polydisperse; i.e. they can have a distribution of possible values. Polydisperse
+parameters are defined as such when the model is coded, and can be activated by
+clicking the *Polydispersity* checkbox on the *Fit Page*.
+
+.. image:: enable_pd.png
+
+Clicking on the *Polydispersity* tab then provides access to these polydisperse
+parameters and allows the type (i.e. the *function* to be used) and 'width'
+(the *PD[ratio]*) to be adjusted. If necessary the 'step size' (*Npts*) and
+'range' (*Nsigs*) of the function can also be adjusted.
+
+.. image:: pd_tab.png
+
+For more information, see the descriptions of :ref:`polydispersityhelp` . In
```

Comment:
Neither here nor the polydispersity docs mention that the distribution is a number density distribution, not a particle volume distribution. This is discussed in [fitting_sq.rst](https://raw.githubusercontent.com/SasView/sasmodels/master/doc/guide/fitting_sq.rst) and can be linked as `PStheory`.

Reply:
Actually, it does say in polydispersity.rst:

![image](https://user-images.githubusercontent.com/10938679/139681356-675b265e-a69c-49b4-b2e2-18a8374c453d.png)

I've now also added a note to the same effect in fitting_help.rst.

Reply:
Probably better to phrase this as `These are all implemented as number densities. The resulting scattering is the number average from the different-sized particles.`, but that would be a sasmodels PR.

Within the current text you may want to mention the ability to reparameterize the particle so that you can control polydispersity according to within-particle constraints (link to plugin.rst). For example, if the process which generates the particles constrains the aspect ratio but not the volume, or if it is a squishy particle that preserves volume but allows a range of aspect ratios for each volume. You may need to define your own distribution function to fully describe the model (link to polydispersity.rst user defined functions).

Reply:
I have fielded several questions from users about the details of the polydispersity calculation, which is why I suggest linking to PStheory in fitting_sq.rst.

This file also discusses the relationship between scale, volume fraction, effective radius, effective radius mode and structure factor mode, which is useful information for the discussion of interaction and mixture models. I think there was an attempt to link to it in on line 105 of this file:
```
Also see :ref:`Interaction_and_Mixture_Models` for further information.
```
but it is self-referential!  Please change it to ref `Interaction_Models`, which is the label used in fitting_sq.rst.

Reply:
Still tweaking the "number-average" phrase. The distributions are implemented as simple distributions, but they are being used to compute the number average. Something like:
```
These distributions define the number density of particles. The resulting scattering is the number average over the distribution.
```
Probably not worth changing.

Reply:
Ok, several changes to polydispersity.rst:

- Have added a link to the end of the opening sentence (that resolves to PStheory): 
![image](https://user-images.githubusercontent.com/10938679/139709727-05be74b1-0242-43ed-a323-ad15b3f776a4.png)
  
- Have changed the 'number distribution' sentence to read:
![image](https://user-images.githubusercontent.com/10938679/139709950-45423ad0-06d2-4660-a958-7ca5706d5480.png)

- In array distribution have changed the existing text after the example array to a Note (the existing text had grammatical & typo errors anyhow) and added a new sentence to the bottom:
![image](https://user-images.githubusercontent.com/10938679/139710106-fb135ded-cb20-4887-a003-e902bb4a6e85.png)

Reply:
And in fitting_help.rst:

- Have changed the link from :ref:`Interaction_and_Mixture_Models` to :ref:`Interaction_Models` as requested.

- Have also added a link to :ref:`PStheory`: from the Polydisperse Parameters sub-section:
![image](https://user-images.githubusercontent.com/10938679/139712617-77634abe-95c8-479b-a2e4-d90ebe691698.png)

- Have revised the wording of the note regarding number density to echo that now in polydispersity.rst:
![image](https://user-images.githubusercontent.com/10938679/139712680-e9f3b3b9-fce3-446c-9869-72e907a504e2.png)

- Added a new note:
![image](https://user-images.githubusercontent.com/10938679/139712886-b2f9c1b9-51cc-46de-a2b2-911c381bb128.png)

- And have added a section flagging that you can reparameterize models:
![image](https://user-images.githubusercontent.com/10938679/139712951-a7da6f69-85ca-49e1-bd3b-9b4173bc3434.png)